solr/core/src/java/org/apache/solr/util/UrlScheme.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.solr.common.cloud;
+package org.apache.solr.util;
```

Comment:
In the backport to 8.11.x, I'll keep `UrlScheme` in Solrj in the `org.apache.solr.common.cloud` package as client apps may be using it. It will be marked as Deprecated and removed from `solrj` in 9.x